provision/roles/ci_nfs/server/files/scriptApproval.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
What does this script do? Can you add some comments at the top of the file to explain its purpose ?

Reply:
done, on PR #32